src/anvil/mod.rs

```diff
@@ -18,14 +17,35 @@ const PORT: ContainerPort = ContainerPort::Tcp(8545);
 ///
 /// The endpoint of the container is intended to be injected into your provider configuration, so that you can easily run tests against a local Anvil instance.
 /// See the `test_anvil_node_container` test for an example of how to use this.
+///
+/// To use the latest Foundry image, you can use the `latest()` method:
+///
+/// ```rust
+/// let node = AnvilNode::latest().start().await?;
+/// ```
+///
+/// Users can use a specific Foundry image in their code with [`ImageExt::with_tag`](https://docs.rs/testcontainers/0.23.1/testcontainers/core/trait.ImageExt.html#tymethod.with_tag).
+///
+/// ```rust
+/// let node = AnvilNode::with_tag("master").start().await?;
```

Comment:
```suggestion
/// ```rust
/// use testcontainers_modules::{anvil::AnvilNode, testcontainers::ImageExt}; 
/// let node = AnvilNode::with_tag("master").start().await?;
```